data/dod_workflow.yaml

```diff
@@ -36,7 +255,16 @@
           human: For any purpose other than to evaluate the proposal, this data except
                  proposal cover sheets shall not be disclosed outside the Government
                  and shall not be duplicated, used or disclosed in whole or in part,
-                 provided that if a contract is awarded blah blah blah blah
+                 provided that if a contract is awarded to this proposer as a result
```

Comment:
Yours is more succinct. I think I'll stick to mine for the timebeing. Perhaps we should do some user research on the issue?